framework/src/eventloop/eventloop_internal.h

```diff
@@ -30,7 +30,15 @@
 
 #define LOOP_IS_STOPPED(loop) (loop->stop_flag == 1)
 
+/* EventLoop async type used for thread-safe call */
+typedef uv_async_t el_async_t;
+
 el_loop_t *get_app_loop(void);
 int get_async_task(void);
 
+/* Unregister Functions for each handles */
+void eventloop_unregister_timer(el_timer_t *);
```

Comment:
Violation of coding guidelines
Use named parameters in function declaration.

return_type function_name(int, char); // not allowed
return_type function_name(int arg1, char arg2); // Use this